README.md

```diff
@@ -32,67 +30,80 @@ This is the original firmware.
 ## Flashing your dryer
 
 > [!CAUTION]
-> Attempting to program your dryer may cause irreparable damage. I make no guarantees. Proceed at your own risk.
+> Attempting to program your dryer may cause irreparable damage if done incorrectly. I'm not liable for any damage you may do to your device. Proceed at your own risk.
 
 You will need:
 
 * A SOVOL SH01 (SH02 may also work - check the chip)
 * A Phillips screwdriver
-* A Windows PC (or VM with access to USB)
```

Comment:
> In regards to why I removed windows; Your instructions should work on any machine running unix. Though someone on Max/Linux machine would need to use WINE for emulation to run HDSC MCU programmer.

@FroggMaster did you use wine on Linux/MacOS?

If so, then please add some more detail about how to run under wine, but I still think that's useful to make it clear that the programmer tool is one designed for Windows (rather than just "PCs")

If not, I suggest reverting that change, because neither of us know whether that will work at all or whether there are any special instructions. I concede that it may be nice to add a paragraph to suggest this possible route and that further investigation is needed.